lib/propolis/src/enlightenment/hyperv/hypercall.rs

```diff
@@ -0,0 +1,81 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Support for hypercalls and their related MSRs.
+
+use crate::common::{GuestAddr, PAGE_SHIFT, PAGE_SIZE};
+
+const LOCKED_BIT: u64 = 1;
+const LOCKED_MASK: u64 = 1 << LOCKED_BIT;
+const ENABLED_BIT: u64 = 0;
+const ENABLED_MASK: u64 = 1 << ENABLED_BIT;
+const GPA_MASK: u64 = !((1 << PAGE_SHIFT) - 1);
```

Comment:
Could use `PAGE_MASK` here, or in place of `GPA_MASK` all together.

Reply:
Done in dffcd6b.